.buildkite/pipeline.yml

```diff
@@ -34,13 +34,16 @@ steps:
   - wait
 
   - label: 'build'
-    command: 'make'
+    commands:
+      - 'make'
+      - 'make -C cni'
```

Comment:
How about invoking `make -C cni` from the top-level make like other targets?

Reply:
I think I prefer them being separate since `tc-redirect-tap` is what's built by `make -C cni` and it's more of an optional part of the Go SDK (and eventually will probably not be in this repo at all).